schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
-);
+  id INT,
+  name VARCHAR(250),
+  date_of_birth date,
+  escape_attempts INT,
+  neutered boolean,
+  weight_kg decimal
+  );
```

Comment:
- [ Optional ] Kindly I recommend you make the id column not null and auto incremented and to mark the id column as primary key.
`id int NOT NULL IDENTITY(1,1), PRIMARY KEY (id)`